arxiv_latex_cleaner/arxiv_latex_cleaner.py

```diff
@@ -526,4 +529,4 @@ def _find_and_replace_patterns(content, patterns_and_insertions):
       content = content[:m.start()] + local_insertion + content[m.end():]
       m = p.search(content)
     logging.info('Finished pattern: %s.', description)
-  return content
+  return content
```

Comment:
Please restore the newline at end of file.

Reply:
Hello @jponttuset,
Thank you and I have updated the code.